.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+Bug Report
+==========
+Check to see if a bug report already exists.  If you still want to create a new issue please include the information below.
+
+**Stack Trace or error message**
+
+
+**Environment**
+* Sia version:
+* OS:
+
+
+**Any useful logs**
+
+
+
+Feature Request
+===============
+Check to see if the feature is on the roadmap.  If it is not on the roadmap then clearly describe your feature request and why it would benefit {{.RepoName}}
```

Comment:
is `{{.RepoName}}` automatically filled in? Is there a reason to not just use Sia?